app/controllers/AmendsController.scala

```diff
@@ -173,24 +173,9 @@ class AmendsController @Inject() (
           throw Exceptions.RequiredValueNotDefinedException("amendmentOutcome", "notificationId")
         }
         if (Constants.amendmentCodesList.contains(id)) {
-          lazy val protectionModel: Future[Either[Result, AmendResponseModel]] =
-            if (id == 7 || id == 14) {
-              sessionCacheService.fetchAndGetFormData[ProtectionModel]("openProtection").map { modelOption =>
-                modelOption
-                  .map { fixedProtectionModel =>
-                    val modelCopy = model.protection.copy(
-                      protectionReference = fixedProtectionModel.protectionReference,
-                      certificateDate = fixedProtectionModel.certificateDate,
-                      protectionType = fixedProtectionModel.protectionType
-                    )
-                    Right(
-                      AmendResponseModel(modelCopy)
-                    )
-                  }
-                  .getOrElse(
-                    Left(throw Exceptions.RequiredValueNotDefinedException("amendmentOutcome", "Active Protection"))
-                  )
-              }
+          val protectionModel: Future[Either[Result, AmendResponseModel]] =
+            if (Constants.fixedProtectionNotificationId.contains(id)) {
```

Comment:
This condition could also be part of the extracted method. Only its name should be different, maybe sth like: `createProtectionModel`.